BLE: Bond Management service & sample (#2551)
nrfconnect/sdk-nrf: samples/bluetooth/peripheral_bms/README.rst

.. _peripheral_bms:

Bluetooth: Peripheral Bond Management Service (BMS)
###################################################

The peripheral BMS sample demonstrates how to use the :ref:`bms_readme`.

Overview
********

When connected, the sample waits for Client requests to perform any bond deleting operation.

It supports up to two simultaneous Client connections.

Requirements
************

> * One of the Nordic development boards:

Reviewer: Include the supported DKs from sample_boardnames.txt

Reviewer 2: fixed

  * |nRF5340DK|,
  * |nRF52840DK|,
  * |nRF52DK|.

* A Bluetooth Low Energy dongle or development kit.

> * nRF Connect for Desktop.

Reviewer: no full stop here or in line 24, these are not sentences (this one is just a product name)

Reviewer 2: fixed

User interface
**************

LED 1:

>    * It blinks with a period of 2 seconds with the duty cycle set to 50% when the main loop is running, and the device is advertising.

Reviewer: ```suggestion
   * Blinks with a period of 2 seconds with the duty cycle set to 50% when the main loop is running and the device is advertising.
```
Also , do not put an asterisk here in these User interface descriptions. These are not supposed to be lists because almost never will there be more than one item here. Follow the format from the template:
https://github.com/nrfconnect/sdk-nrf/pull/2632/files (around line 95)

Reviewer 2: fixed

LED 2:

>    * It is on when connected.

Reviewer: ```suggestion
   * On when connected.
```

Reviewer 2: fixed

Building and Running
********************
.. |sample path| replace:: :file:`samples/bluetooth/peripheral_bms`

.. include:: /includes/build_and_run.txt

.. _peripheral_bms_testing:

Testing
=======

> After programming the sample to your development board, test it by performing the following steps:

Reviewer: ```suggestion
After programming the sample to your development kit, test it by performing the following steps:
```
(We avoid calling it 'board' now)

Reviewer 2: fixed

1. |connect_terminal_specific|
#. Reset the board.
#. Start `nRF Connect for Desktop`_ and select the connected device that is used for communication.
#. Connect to the device from nRF Connect.
   The device is advertising as "Nordic_BMS".

> #. Bond to the device.

Reviewer: ```suggestion
#. Bond with the device.
```

Reviewer 2: fixed

>    To bond:

Reviewer: remove this line, end the previous one with a colon

Reviewer 2: fixed

>    a. click the settings button for the device in nRF Connect,

Reviewer: Better make these points into standalone sentences with capital letters and full stops.
This structure with commas and fulls stop after only the last point is good when you want to enumerate some items, or for example to present different options. But when it comes to actual steps or tasks in a procedure, it should definitely be standalone sentences.

Reviewer 2: ok, will keep that in mind for the next PRs. Fixed

>    b. select **Pair**,

Reviewer: Use the :guilabel: element instead of bold when referring to UI items (try to use semantic tagging whenever possible). So:
```suggestion
   b. Select :guilabel:`Pair`.
```

Reviewer 2: fixed

   c. check **Perform Bonding**,
   d. click **Pair**.
   

> #. Verify in the logs that the connection security is updated.

Reviewer: ```suggestion
#. Check the logs to verify that the connection security is updated.
```

Reviewer 2: fixed
